pkg/agent/openflow/client.go

```diff
@@ -326,6 +329,15 @@ type Client interface {
 	// installed by InstallMulticastGroup.
 	UninstallMulticastGroup(groupID binding.GroupIDType) error
 
+	// InstallMulticastFlexibleIPAMGroups install two groups when flexibleIPAM is enabled.
+	// One group matches the multicast packets received from the uplink interface,
+	// with one Bucket forcing the packet to go into OVS multicast pipeline
+	// and the other outputting the traffic to the host interface.
+	// The other group matches the multicast packets received from the host interface,
+	// with one Bucket forcing the packet to go into OVS multicast pipeline
+	// and the other outputting the traffic to the uplink interface.
+	InstallMulticastFlexibleIPAMGroups(inboundGroupID binding.GroupIDType, outboundGroupID binding.GroupIDType) error
```

Comment:
I feel it's better to add a few lines in the commit and PR summary about the new flows and what is added to support FlexibleIPAM in Multicast, so reviewers can understand the change better.

Reply:
updated